dist/emoji-en-US.json

```diff
@@ -9887,12 +9921,8 @@
     "pride",
     "gay",
     "lgbt",
-    "glbt",
     "queer",
-    "homosexual",
-    "lesbian",
```

Comment:
why was this removed?

Reply:
- "glbt" is not a term that's used in the queer community, and is used as a dogwhistle to downplay women's rights. https://theforeword.org/832/editorials/the-l-in-lgbt-and-why-order-matters/ 
- the pride flag is not the same as the gay man flag or the lesbian flag.

Reply:
Sorry, I just meant lesbian specifically. Is it in the same vein that gay is preserved but not lesbian?

Reply:
Ah sorry I missed this reply. Generally, 🏳️‍🌈 is referred to as just the pride flag (although it started out as the gay flag). Although queer/LGBTQ pride obviously includes lesbian, the 🏳️‍🌈 flag isn't the lesbian flag and thus shouldn't be aliased to "lesbian".

Reply:
considering that there’s not a lesbian flag in emoji, do you mind if we keep this?

Reply:
The lesbian flag is often implemented w/ a ZWJ or custom emoji.

Reply:
IMO given the fact that neither gay man flag and nor lesbian flag is in unicode, I think erring on being inclusive of the broader definition for pride is my preference so that there is at least something to be found.